Scripts/files.py

```diff
@@ -553,7 +553,7 @@ def choose_config_file(configDict, newestConfigVersion):
   configFileList = list_config_files()
   # If only one config file exists, prompt to use
   if len(configFileList) == 0:
-    if choice(f"\nFound {F.YELLOW}config file{S.R}, use those settings?") == False:
+    if choice(f"Found {F.YELLOW}config file{S.R}, use those settings?") == False:
```

Comment:
why no `addCancel=True` for this function call?

Reply:
I believe that the `None` case isn't handled, if you choose cancel the program should quit without logging you out

Reply:
i think it is bug from the program itself, which can be fixed on another pr

after looking around i found choice function called 7x with `addCancel=True` and 1x without that paramater

so i think addCancel is not useful